README.md

```diff
@@ -124,6 +124,21 @@ base_uri: "https://appcanary.com/api/v3"
 monitor_name: "my_monitor"
 ```
 
+### Heroku
+
+For Heroku rails deployments, everything should really "just work".
+
+1. Include the `appcanary` gem in your `Gemfile`.
+2. Ensure you have the `api_key` set to `ENV["APPCANARY_API_KEY"]`.
```

Comment:
By following the instructions above this section, outlining all the ways you can configure the gem. Should I re-iterate all that here?